apps/issuer/models.py

```diff
@@ -227,6 +227,8 @@ class Issuer(ResizeUploadedImage,
         'country': None
     }
 
+    __original_verified = False
```

Comment:
It is not clear what this variable is or does, so it would be nice to document it with a comment